ui/page/send/page.go

```diff
@@ -554,6 +597,29 @@ func (pg *Page) HandleUserInteractions() {
 	}
 }
 
+// Handle is called just before Layout() to determine if any user
+// interaction recently occurred on the modal and may be used to update the
+// page's UI components shortly before they are displayed.
+func (pg *Page) Handle() {
+	pg.HandleUserInteractions()
+
+	if pg.modalLayout.BackdropClicked(true) {
+		pg.modalLayout.Dismiss()
+	}
```

Comment:
If you're going to dismiss the modal, no need to call `HandleUserInteractions` first.
```suggestion
	if pg.modalLayout.BackdropClicked(true) {
		pg.modalLayout.Dismiss()
	} else {
		pg.HandleUserInteractions()
	}
```